parts/linux/cloud-init/artifacts/mariner/cse_install_mariner.sh

```diff
@@ -39,14 +39,26 @@ installDeps() {
     done
 
     # install 2.0 specific packages
-    # apparmor related packages and the blobfuse package are not available in AzureLinux 3.0
+    # the blobfuse package is not available in AzureLinux 3.0
     if [ "$OS_VERSION" = "2.0" ]; then
       for dnf_package in apparmor-parser libapparmor blobfuse; do
         if ! dnf_install 30 1 600 $dnf_package; then
           exit $ERR_APT_INSTALL_TIMEOUT
         fi
       done
     fi
+
+    # install apparmor related packages in AzureLinux 3.0
+    # apparmor-utils is not installed in VHD as it brings auditd dependency
```

Comment:
Installing `apparmor-utils` brings in the `audit` package - Hence, the [testAuditDNotPresent](https://github.com/Azure/AgentBaker/blob/master/vhdbuilder/packer/test/linux-vhd-content-test.sh#L497C1-L507C2) linux vhd content test starts failing.